### ISSUE TITLE
Documentation Feature - Hidden Field

### ISSUE THREAD

**Issue by danielerne** [dachcom-digital/pimcore-formbuilder#270]
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | yes
| BC Break report? | no
| RFC?             | no

There should be a documentation how to populate a hidden field with a value. Is it the correct way to do it with the formbuilder post data event over custom options? And how is it possible to render it in the mail template?

**Comment by christopher-siegel** [comment]
You can put the value in the field "Preselected". Not sure why its called that way.

![Bildschirmfoto 2021-03-30 um 15 40 34](https://user-images.githubusercontent.com/2526628/112998381-47aecc00-916e-11eb-9211-07a182209a82.png)

**Comment by solverat** [comment]
@christopher-siegel correct. And you're also right about the label - i guess we've started with this `data` (That's how symfony calls this attribute) element within the choice element and never thought about the naming in other context!
